### ISSUE TITLE
nvda does not correctly identify the password field when creating a new email account in outlook

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8380]
### Steps to reproduce:
Create a new email account in Outlook.
In the field where you must enter the account password, nvda does not correctly identify this field.
### Expected behavior:
The nvda should inform that this is a protected edit edit field of password, when typing the characters of the password the nvda should not announce the characters typed and when navigating with the arrows by the typed password text must inform the asterisk.

### Actual behavior:
nvda does not advertise that the text box is to enter a password, protected field, password edit.
When typing the characters, nvda announces the characters you typed.
When navigating with the arrows by the typed text the nvda gives a sound of error and does not announce asterísco.

### System configuration:
NVDA version:
next 15211
NVDA Installed or portable:
instaled
Other information:
do the tests in a version of office 2016 updated, it seems that in office 2016 with the latest update the account creation screen has changed.

Windows version:
windows 10 17134.81
Name and version of other software in use when reproducing the issue:
outlook 2016 16.0.9330.2087
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
log:
[outlook.txt](https://github.com/nvaccess/nvda/files/2085715/outlook.txt)

**Comment by fernando-jose-silva** [comment]
this behavior is also present when doing the tests in an updated software suite:
nvda alfa 16733
windows 10 17763.292
Microsoft Office version 16.0.11231.20130

**Comment by michaelDCurran** [comment]
This is a limitation in older versions of Outlook. It does not expose the protected state on its password fields. There is nothing we can do. Microsoft has addressed this in Outlook 2016.
